docs/platform/core/syntax/basic-syntax.mdx

```diff
@@ -118,4 +119,14 @@ actions:
 
 The `provider` configuration is already covered in [Providers](syntax#provider)
 
-_*** Actions are the last section of an alert_
\ No newline at end of file
+### On-failure
+```yaml
+on-failure:
+    # Just need a provider we can use to send the failure reason
+    provider:
+      type: slack
+      config: " {{ providers.slack-demo }} "
```

Comment:
add the mesage ("Alert failed to run")

Reply:
It's being created when the exception occurs. I'll refactor this soon so merging for now.